exercises/leap/metadata.toml

```diff
@@ -3,3 +3,4 @@ blurb = "Determine whether a given year is a leap year."
 source = "CodeRanch Cattle Drive, Assignment 3"
 source_url = "https://coderanch.com/t/718816/Leap"
 deep_dive_youtube_id = "Fj5m16is5hI"
+deep_dive_blurb = "Deep dive into the different algorithms and approaches you can use to solve Leap, as Jeremy and Erik explore interesting Community Solutions across 10 different languages on Exercism."
```

Comment:
```suggestion
deep_dive_blurb = "Deep dive into the different algorithms and approaches you can use to solve Leap. Jeremy and Erik explore interesting Community Solutions across 10 different languages on Exercism."
```

Reply:
The reverse string deep dive is a bit more enticing I feel:

> Explore 14 different ways to reverse a string, exploring a range of topics including Unicode codepoints, graphemes, stack vs heap allocations, and pointers. Kick back and enjoy 45 minutes of learning with Jeremy and Erik."

Maybe we could try something here too? 

Another difference between the two is that this deep dive refers to the exercise, whereas the `reverse-string` one explains what is being done (reversing a string). Maybe we can choose one or the other? I think the reverse-string one, where we explain _what_ is being done is more interesting to read.

Reply:
I think adding some examples (like in the reverse-string one) could help, like: "using built in functionality, boolean expressions, modulo operations". Something like that